Marlin/src/feature/leds/neopixel.cpp

```diff
@@ -55,18 +55,19 @@ void setup_neopixel() {
   set_neopixel_color(pixels.Color(0, 0, 0, 255));    // white
 }
 
-bool neopixel_set_led_color(const uint8_t r, const uint8_t g, const uint8_t b, const uint8_t w, const bool isSequence) {
+bool neopixel_set_led_color(const uint8_t r, const uint8_t g, const uint8_t b, const uint8_t w, const uint8_t p) {
   const uint32_t color = pixels.Color(r, g, b, w);
-  static uint16_t nextLed = 0;
-  if (!isSequence)
+  pixels.setBrightness(p);
+  #if !ENABLED(NEOPIXEL_IS_SEQUENTIAL)
```

Comment:
Here it makes the best sense to do the `#if ENABLED(...)` case and put `return false` at the end.